Gaikan/UIKit/Layer/BorderLayer.swift

```diff
@@ -27,19 +46,19 @@ class BorderLayer : CALayer {
             case .top:
                 self.draw(border: border,
                           path: path,
-                          points: [ CGPoint(x: 0, y: 0), CGPoint(x: ctx.width, y: 0), CGPoint(x: ctx.width, y: Int(border.width)), CGPoint(x: 0, y: Int(border.width)) ])
+                          points: [ CGPoint(x: 0, y: 0), CGPoint(x: bounds.width, y: 0), CGPoint(x: bounds.width, y: border.width), CGPoint(x: 0, y: border.width) ])
```

Comment:
Thing is that ctx depends on screen scale. So, if `layer.contentsScale` is 2.0 (for instance), `ctx.width` is twice the bound width.